test/e2e_broker/broker_trigger_test.go

```diff
@@ -1,273 +0,0 @@
-//go:build e2e
-// +build e2e
-
-/*
- * Copyright 2020 The Knative Authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package e2e_broker
-
-import (
-	"context"
-	"fmt"
-	"testing"
-
-	. "github.com/cloudevents/sdk-go/v2/test"
-
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	cloudevents "github.com/cloudevents/sdk-go/v2"
-	"github.com/google/uuid"
-	"knative.dev/eventing-kafka-broker/control-plane/pkg/kafka"
-	"knative.dev/eventing-kafka-broker/control-plane/pkg/reconciler/broker"
-	testingpkg "knative.dev/eventing-kafka-broker/test/pkg"
-	brokertest "knative.dev/eventing-kafka-broker/test/pkg/broker"
-	kafkatest "knative.dev/eventing-kafka-broker/test/pkg/kafka"
-	eventing "knative.dev/eventing/pkg/apis/eventing/v1"
-	testlib "knative.dev/eventing/test/lib"
-	"knative.dev/eventing/test/lib/recordevents"
-	"knative.dev/eventing/test/lib/resources"
-)
-
-func TestBrokerTrigger(t *testing.T) {
```

Comment:
Didn't migrate this test, as this seems to test only default values and the trigger test should be covered by the conformance tests. The other test (`TestBrokerWithConfig`), tests the same with custom values